react-client/src/components/PlayerEnterNameScreen.jsx

```diff
@@ -5,42 +5,19 @@ class PlayerEnterNameScreen extends React.Component {
   constructor(props) {
     super(props);
 
-  }
+  };
 
   render() {
 
     return (
       <div>
 
-        <h2> Join a Game of Defintely Not Avalon </h2>
-
-
-      I do not presently know how to deal with the html form
-
-     
-<form >
-  <input type="text" name="Token" value="Enter Access Token"/>
-
-  <input type="text" name="Name" value="Enter name"/>
-
-  <input type="submit" value="Submit"/>
-</form> 
-
-      
-
-
-        <button onClick={this.props.backButtonClickHandler}>
-        {'Back'}
-        </button>
-
-        <button onClick={this.props.joinButtonClickHandler}>
-        {'Join'}
-        </button>
-
```

Comment:
I don't mean this in a proprietary way. But, I don't understand 1) why your db functions is touching the render method of a react component nor 2) why this change is a good change.